### DISCUSSION TITLE
New Release (v1.44.0)?

### DISCUSSION THREAD

**Discussion by timkral** [golangci/golangci-lint#2478] [Q&A]
Hi,

I'm interested in using a couple of the new features added since the last official release. I know that your policy is to do a feature based release, not a schedule based one. So just wondering if we're close to a v1.44.0 release from a feature perspective.

https://github.com/golangci/golangci-lint/compare/v1.43.0...master

Thanks.
-Tim

**Comment by mgjules** [comment]
I'm also interested since `v1.43.0` is quite dated now and need to use the new features.
